app/email.go

```diff
@@ -321,14 +321,23 @@ func (a *App) SendInviteEmails(team *model.Team, senderName string, senderUserId
 		return
 	}
 
+	sender, appErr := a.GetUser(senderUserId)
+	if appErr != nil {
+		a.Log().Error("Email invite not sent, unable to find the sender user.",
+			mlog.String("user_id", senderUserId),
+			mlog.String("team_id", team.Id),
+			mlog.Err(appErr))
+		return
+	}
+
 	for _, invite := range invites {
 		if len(invite) > 0 {
 			subject := utils.T("api.templates.invite_subject",
 				map[string]interface{}{"SenderName": senderName,
 					"TeamDisplayName": team.DisplayName,
 					"SiteName":        a.ClientConfig()["SiteName"]})
 
-			bodyPage := a.newEmailTemplate("invite_body", model.DEFAULT_LOCALE)
+			bodyPage := a.newEmailTemplate("invite_body", sender.Locale)
```

Comment:
Isn't this needs to be something like server default locale? I am not sure if this should be sender locale. The sender and receiver may speak totally different languages..

Reply:
Yep, I totally agree. That's what I initially thought. But the thing is that the rest of the mail is already translated, as you can see with the other props after this line. It's just the footer which wasn't. See [here](https://github.com/mattermost/mattermost-server/issues/14809), everything was already being translated. So I went forward with the implementation.

Reply:
I see, haven't looked the rest of the code. Thanks for the clarification.